app/scripts/lib/typed-message-manager.js

```diff
@@ -221,7 +238,7 @@ module.exports = class TypedMessageManager extends EventEmitter {
     msg.status = status
     this._updateMsg(msg)
     this.emit(`${msgId}:${status}`, msg)
-    if (status === 'rejected' || status === 'signed') {
+    if (status === 'rejected' || status === 'signed' || status === 'errored') {
```

Comment:
unclear when status gets set to 'errored' due to errors thrown in validateParams

Reply:
I'm not sure I follow; what's unclear? The conditions that cause a message to be considered invalid? Error descriptions should be fine; are you suggesting to improve the documentation for the `validateParams` method to explicitly state what conditions are validated?

Reply:
Ah sorry I was being dense, I didn't see the call to set the message status in metamask-controller